modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/Checkbox.es.js

```diff
@@ -156,6 +156,7 @@ const Main = ({
 				systemSettingsURL={systemSettingsURL}
 				visible={visible}
 			/>
+			<ClayInput name={name} type="hidden" value={checked} />
```

Comment:
@marcelabc, can you please give more details about how are you getting this value in the back-end?

Thanks.

Reply:
When submitting a value to the boolean field in a published form, the value of the "ddmFormFieldParameterName" was null in the request ([CheckboxDDMFormFieldValueRequestParameterRetriever.java](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/CheckboxDDMFormFieldValueRequestParameterRetriever.java)). I think that the reason is that there was no input saving/updating the value.

All the input hidden is serialized automatically to the request. I created one with the same name of the field, so can be found in the httpServletRequest in the backend. I don't know the specific details, but other fields has this kind of input as well (example: [Radio.es.js)](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Radio/Radio.es.js)